extensions/amp-analytics/0.1/test/test-visibility-model.js

```diff
@@ -456,37 +463,75 @@ describes.sandboxed('VisibilityModel', {}, () => {
       let reportPromise;
       let promiseResolver;
       beforeEach(() => {
-        reportPromise = new Promise(resolve => {
-          promiseResolver = resolve;
-        });
-        vh.setReportReady(() => {return reportPromise;});
+        promiseResolver = new Deferred();
```

Comment:
nit: naming, is it possible to either rename `promiseResolver`, or make it the resolver function?